src/dsc/Microsoft.PowerToys.Configure/examples/configureLauncherPlugins.dsc.yaml

```diff
@@ -0,0 +1,17 @@
+properties:
+  resources:
+    - resource: PowerToysConfigure
+      directives:
+        description: Configure PowerToys
+      settings:
+        Debug: false
+        PowerLauncher:
+          Enabled: true
+          AdditionalProperties:
+            Plugins:
+              - Name: "Calculator"
+                Disabled: true
```

Comment:
Do we only support disabled state?

Reply:
we support both states, that's just naming of the plugin properties here.

Reply:
Unclear asked. Is this the only setting you can set?

Reply:
I think we should support other options as well. Not the nested objects though. I'll compose a list of settable options for all modules and create a PR in https://github.com/MicrosoftDocs/windows-dev-docs/blob/docs/hub/powertoys/index.md